Client/Frontend/Browser/TabManager.swift

```diff
@@ -112,6 +112,7 @@ class TabManager : NSObject {
         addNavigationDelegate(self)
 
         NSNotificationCenter.defaultCenter().addObserver(self, selector: #selector(TabManager.prefsDidChange), name: NSUserDefaultsDidChangeNotification, object: nil)
+        NSNotificationCenter.defaultCenter().addObserver(self, selector: #selector(TabManager.noImageModePrefDidChange), name: NoImageModeStatusChangedNotification, object: nil)
```

Comment:
You do the work of `noImageModePrefDidChange` in `TabManager.prefsDidChange` rather than using a separate event (i.e. you don't need a `NoImageModeStatusChangedNotification`).